src/common/makeWclApiUrl.ts

```diff
@@ -1,12 +1,13 @@
 import makeApiUrl from './makeApiUrl';
+import { QueryParams } from './QueryParams';
 
-const WCL_API_KEY = process.env.REACT_APP_WCL_API_KEY || undefined;
-if (!WCL_API_KEY && process.env.NODE_ENV === 'development') {
```

Comment:
You are changing the behavior here in such a way that I think this would now break in production. I am pretty sure the development check is required.

Reply:
If there's no key, then it would be on development. But even if you pass an empty string, it would still be defined as a string and not an undefined. This would still break production, if it was an empty key. Right?

Reply:
The webapp used by the production environment also omits the API key because it is the only way to keep it a secret. This check ensures the app still works in production with that setup.

The env var `REACT_APP_WCL_API_KEY` could be omitted entirely, which would make it `undefined`. It could also be left to an empty value like `REACT_APP_WCL_API_KEY=` (that being the entire line), which would make it an empty string.

Reply:
Alright! Thanks for teaching me some new stuff. 👍 Guess we will disregard those changes in makeWclApiUrl.js then. Is there any other issues with the other contributions I've done?